### ISSUE TITLE
Google search with results from 13ways - tidying up

### ISSUE THREAD

**Issue by vns2** [janastu/ncbs-modular#255]
The 13 ways site is crawled by Google. And so it comes up on early search result. Here's an example. I think we have to think about how it looks when searched. This, for instance, is pulling up the Omeka backend page, and the search result is full of html code. 

![google_search_result](https://user-images.githubusercontent.com/4581081/33306905-84d77784-d43a-11e7-987e-c95f41f06d0a.png)

**Comment by salus-sage** [comment]
Yes, right now the exhibit is not indexed and searched by crawlers, so it
won't come on search results.

**Comment by salus-sage** [comment]
Required solutions : 
Update ncbs25 production,  Dev omeka robots. Txt to no crawling.

**Comment by salus-sage** [comment]
https://support.archive-it.org/hc/en-us/articles/208001096-Avoid-robots-exclusions?mobile_site=true
